temporal-sdk/src/main/java/io/temporal/worker/Worker.java

```diff
@@ -493,12 +503,22 @@ private static SingleWorkerOptions toWorkflowWorkerOptions(
       Scope metricsScope) {
     Map<String, String> tags =
         new ImmutableMap.Builder<String, String>(1).put(MetricsTag.TASK_QUEUE, taskQueue).build();
+
+    Duration stickyQueueScheduleToStartTimeout = options.getStickyQueueScheduleToStartTimeout();
+    if (WorkerOptions.DEFAULT_STICKY_SCHEDULE_TO_START_TIMEOUT.equals(
+            stickyQueueScheduleToStartTimeout)
```

Comment:
Didn't read the rest of the code, but assuming this is "if default fallback" code, any concerns of a user explicitly setting the same timeout as our default that we will now override?

Reply:
Yeah. For this to "fire", user needs to set an option on the Workers into the default value AND set a DIFFERENT value on WorkerFactory too using the deprecated method. Not a likely scenario to make something special about WorkerOptions#stickyQueueScheduleToStartTimeout and incur maintenance costs in the future.